applications/smart-distancing/eval/Dockerfile-eval

```diff
@@ -0,0 +1,23 @@
+# docker can be installed by following instructions: 
+# build: docker build -f Dockerfile-amd64usbtpu-eval -t edgetpu-detector-eval .
+# run: docker run -it -v /PATH_TO_CLONED_REPO_ROOT/:/repo detector-eval
```

Comment:
please provide executable build and run commands and relevant tag for docker. 
how about this?
neuralet/tools:detector-eval or neuralet/edgetpu:detector-eval

Reply:
Done